File: it.unibo.cautiousExplorer/app/src/main/java/it/unibo/remoteCall/TripInfo.java
/*
============================================================
TripInfo

============================================================
 */
package it.unibo.remoteCall;

import mapRoomKotlin.RoomMap;
import mapRoomKotlin.mapUtil;

public class TripInfo {
    private RoomMap map = RoomMap.Companion.getRoomMap();   //init
    private String journey = "";

    public void updateMovesRep(String move){
        mapUtil.doMove( move );
        journey = journey + move;
    }

    public void setObstacle(){ mapUtil.setObstacle(); }
    public String getJourney(){ return journey; }
    public String getMap(){ return mapUtil.getMapRep(); }
    public String getDirection(){ return mapUtil.getDirection(); }
    public void showMap(){ mapUtil.showMap(); }
    public void showJourney(){ System.out.println(journey); }

}

File: it.unibo.cautiousExplorer/backup/mapRoomKotlinOld/TripInfo.kt
package mapRoomKotlin

import mapRoomKotlin.mapUtil.getMapRep

class TripInfo( ) {

    private var journey = ""
    var map   = RoomMap.getRoomMap()

    fun updateMovesRep(move: String) {
        mapUtil.doMove(move)
        journey = journey + move
    }

    fun setObstacle() {
        mapUtil.setObstacle()
    }

    fun getJourney(): String {
        return journey
    }

    fun getMap(): String {
        return getMapRep()
    }

    fun getDirection(): String {
        return mapUtil.getDirection()
    }

    fun showMap() {
        mapUtil.showMap()
    }

    fun showJourney() {
        println(journey)
    }


}